frontend/src/components/Balloon.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  useState,
+} from "react";
+import styled from "styled-components";
+import { BALLOON } from "../constants/images";
+
+interface BallonProps {
+  top?: number;
+  left?: number;
+  onPop?: any;
```

Comment:
before I merge if I get approved, I honestly don't know what to put for this. I tried doing either void or () => void, both gave an error in either Balloon.tsx or Reward.tsx...

Reply:
I think you should have it as `(void) => void;`. I _thought_ `() => void;` would work, but I certainly could be wrong